OpenRA.Mods.Common/Traits/Husk.cs

```diff
@@ -43,10 +43,13 @@ IEnumerable<object> IActorPreviewInitInfo.ActorPreviewInits(ActorInfo ai, ActorP
 		bool IOccupySpaceInfo.SharesCell { get { return false; } }
 	}
 
-	public class Husk : IPositionable, IFacing, ISync, INotifyCreated, INotifyAddedToWorld, INotifyRemovedFromWorld, IDeathActorInitModifier
+	public class Husk : IPositionable, IFacing, ISync, INotifyCreated, INotifyAddedToWorld, INotifyRemovedFromWorld,
+		IDeathActorInitModifier, IEffectiveOwner
 	{
-		readonly HuskInfo info;
 		readonly Actor self;
+		readonly bool isDisguised;
```

Comment:
I'd prefer if this was made less ambigous, so nobody confuses this with disguise logic for actors.
Something like `effectiveOwnerDisguised`, even if it's a bit long.